code/modules/mob/mob_defines.dm

```diff
@@ -65,9 +65,6 @@
 	///Cursor icon used when holding shift over things
 	var/examine_cursor_icon = 'icons/effects/mouse_pointers/examine_pointer.dmi'
 
-	///Whether this mob has or is in the middle of committing suicide.
-	var/suiciding = FALSE
-
 	/// Whether a mob is alive or dead. TODO: Move this to living - Nodrak (2019, still here)
```

Comment:
half tempted to write (2023, still here) but

Reply:
I'd rather you replace it with "Ideally this is on living instead, but an enormous amount of code would need to change"

Reply:
I'll chalk it up to "out of scope for the purposes of this PR", just think it's funny that this var was originally put in such close proximity to this